packages/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -148,6 +150,9 @@ describe('plugin-mercury', () => {
           assert.isTrue(mercury.connected, 'Mercury is connected');
           assert.isFalse(mercury.connecting, 'Mercury is not connecting');
           assert.calledWith(socketOpenStub, sinon.match(/ws:\/\/example.com/), sinon.match.any);
+          mercury._emit('event:featureToggle_update', updatedFeature)
+          assert.calledOnce(webex.internal.feature.updateFeature);
+          assert.calledWith(webex.internal.feature.updateFeature, updatedFeature);
```

Comment:
Can we not user assert.calledOncewith() here?

Reply:
There was no `calledOnceWith`

However found a `calledOnceWithExactly` and changed